boards.txt

```diff
@@ -22338,6 +22338,238 @@ m5stack_atoms3.menu.EraseFlash.all.upload.erase_cmd=-e
+
+m5stack_atoms3u.menu.FlashMode.qio=QIO 80MHz
+m5stack_atoms3u.menu.FlashMode.qio.build.flash_mode=dio
+m5stack_atoms3u.menu.FlashMode.qio.build.boot=qio
+m5stack_atoms3u.menu.FlashMode.qio.build.boot_freq=80m
+m5stack_atoms3u.menu.FlashMode.qio.build.flash_freq=80m
+m5stack_atoms3u.menu.FlashMode.qio120=QIO 120MHz
+m5stack_atoms3u.menu.FlashMode.qio120.build.flash_mode=dio
+m5stack_atoms3u.menu.FlashMode.qio120.build.boot=qio
+m5stack_atoms3u.menu.FlashMode.qio120.build.boot_freq=120m
+m5stack_atoms3u.menu.FlashMode.qio120.build.flash_freq=80m
+m5stack_atoms3u.menu.FlashMode.dio=DIO 80MHz
+m5stack_atoms3u.menu.FlashMode.dio.build.flash_mode=dio
+m5stack_atoms3u.menu.FlashMode.dio.build.boot=dio
+m5stack_atoms3u.menu.FlashMode.dio.build.boot_freq=80m
+m5stack_atoms3u.menu.FlashMode.dio.build.flash_freq=80m
+m5stack_atoms3u.menu.FlashMode.opi=OPI 80MHz
+m5stack_atoms3u.menu.FlashMode.opi.build.flash_mode=dout
+m5stack_atoms3u.menu.FlashMode.opi.build.boot=opi
+m5stack_atoms3u.menu.FlashMode.opi.build.boot_freq=80m
+m5stack_atoms3u.menu.FlashMode.opi.build.flash_freq=80m
+
+
+m5stack_atoms3u.menu.FlashSize.8M=8MB (64Mb)
+m5stack_atoms3u.menu.FlashSize.8M.build.flash_size=8MB
+
+m5stack_atoms3u.menu.LoopCore.1=Core 1
+m5stack_atoms3u.menu.LoopCore.1.build.loop_core=-DARDUINO_RUNNING_CORE=1
+m5stack_atoms3u.menu.LoopCore.0=Core 0
+m5stack_atoms3u.menu.LoopCore.0.build.loop_core=-DARDUINO_RUNNING_CORE=0
+
+m5stack_atoms3u.menu.EventsCore.1=Core 1
+m5stack_atoms3u.menu.EventsCore.1.build.event_core=-DARDUINO_EVENT_RUNNING_CORE=1
+m5stack_atoms3u.menu.EventsCore.0=Core 0
+m5stack_atoms3u.menu.EventsCore.0.build.event_core=-DARDUINO_EVENT_RUNNING_CORE=0
+
+m5stack_atoms3u.menu.USBMode.hwcdc=Hardware CDC and JTAG
+m5stack_atoms3u.menu.USBMode.hwcdc.build.usb_mode=1
+m5stack_atoms3u.menu.USBMode.default=USB-OTG (TinyUSB)
+m5stack_atoms3u.menu.USBMode.default.build.usb_mode=0
+
+m5stack_atoms3u.menu.CDCOnBoot.cdc=Enabled
+m5stack_atoms3u.menu.CDCOnBoot.cdc.build.cdc_on_boot=1
+m5stack_atoms3u.menu.CDCOnBoot.default=Disabled
+m5stack_atoms3u.menu.CDCOnBoot.default.build.cdc_on_boot=0
+
+m5stack_atoms3u.menu.MSCOnBoot.default=Disabled
+m5stack_atoms3u.menu.MSCOnBoot.default.build.msc_on_boot=0
+m5stack_atoms3u.menu.MSCOnBoot.msc=Enabled (Requires USB-OTG Mode)
+m5stack_atoms3u.menu.MSCOnBoot.msc.build.msc_on_boot=1
+
+m5stack_atoms3u.menu.DFUOnBoot.default=Disabled
+m5stack_atoms3u.menu.DFUOnBoot.default.build.dfu_on_boot=0
+m5stack_atoms3u.menu.DFUOnBoot.dfu=Enabled (Requires USB-OTG Mode)
+m5stack_atoms3u.menu.DFUOnBoot.dfu.build.dfu_on_boot=1
+
+m5stack_atoms3u.menu.UploadMode.default=UART0 / Hardware CDC
+m5stack_atoms3u.menu.UploadMode.default.upload.use_1200bps_touch=false
+m5stack_atoms3u.menu.UploadMode.default.upload.wait_for_upload_port=false
+m5stack_atoms3u.menu.UploadMode.cdc=USB-OTG CDC (TinyUSB)
+m5stack_atoms3u.menu.UploadMode.cdc.upload.use_1200bps_touch=true
+m5stack_atoms3u.menu.UploadMode.cdc.upload.wait_for_upload_port=true
```

Comment:
The board have only 1 connector (USB), please use only proper settings, also with other stuff like CDC and USB mode